meetups/brno.yml

```diff
@@ -0,0 +1,59 @@
+- name: Začátečnický kurz <br> Jarní pondělí 2017
+  start: 2017-02-27
+  end: 2017-05-29
+  time: pondělky 18:00 – 20:00
+  place:
+    name: Odbor školství
+    address: Cejl 73, Brno
+    latitude: '49.1992294'
+    longitude: '16.6235206'
+  registration:
+    url: https://docs.google.com/forms/d/e/1FAIpQLSczcYKk3vbjd6iYTc9uBbM3s_qEfQzIi-2M0nAGWdbK97FvDA/viewform?c=0&w=1
+
+- name: Začátečnický kurz <br> Jarní úterý 2017
+  start: 2017-02-28
+  end: 2017-05-30
+  time: úterky 18:00 – 20:00
+  place: &experis
+    name: Experis
+    address: BC Titanium, Nové sady 25, Brno
+    latitude: '49.1875653'
+    longitude: '16.6064981E'
+  registration:
+    url: https://docs.google.com/forms/d/e/1FAIpQLSczcYKk3vbjd6iYTc9uBbM3s_qEfQzIi-2M0nAGWdbK97FvDA/viewform?c=0&w=1
+
+- name: Začátečnický kurz <br> Podzim 2016
+  start: 2016-09-26
+  end: 2017-01-09
+  time: pondělky 18:00 – 20:00
+  place: *experis
```

Comment:
ja doplnila, ale resim git a jak to poslat na tvou vetev a vzhledem k tomu, ze me ignorujes na slacku, tak mi to fakt trva :D :D  vyckej uz to asi bude